design-at-cornell/src/components/HomepageStyles.ts

```diff
@@ -1,91 +1,110 @@
+import { Link } from 'react-router-dom';
 import styled from 'styled-components';
+import homepage_banner from '../static/images/homepage_banner.svg';
 
-export const Headline = styled.div`
+export const BannerBackground = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 550px;
+  background-image: url(${homepage_banner});
+  padding-left: 100px;
+  padding-right: 100px;
+  margin-bottom: 90px;
+`;
+
+export const BlurbContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+  align-items: left;
+  justify-content: center;
+  text-align: left;
+  h3 {
+    height: 72px;
+    font-family: 'Work Sans';
+    font-style: normal;
+    font-weight: bolder;
+    font-size: 30px;
+    line-height: 47px;
+    padding-left: 50px;
+  }
+
+  p {
+    font-size: 20px;
+    line-height: 28px;
+    color: black;
+    padding-left: 50px;
+    font-weight: 500;
+  }
+`;
+
+export const Blurb = styled.div`
   text-align: center;
+  display: block;
+  justify-content: center;
+  position: absolute;
+  width: 497px;
+  height: 348px;
+  right: 120px;
+  top: 190px;
+  background: rgba(255, 255, 255, 0.85);
+  border-radius: 20px;
+`;
+
+export const Headline = styled.div`
   font-family: 'Work Sans';
   font-style: normal;
-  font-size: large;
+  font-size: x-large;
   text-align: center;
   color: black;
+  justify-content: center;
 `;
 
 export const MissionStat = styled.div`
   text-align: center;
+  justify-content: center;
   font-family: 'Work Sans';
   font-style: normal;
-  font-size: medium;
-  text-align: center;
-  color: black;
-  padding-top: 20px;
-`;
-
-export const Title = styled.div`
-  font-weight: bold;
-  font-size: 35px;
-  line-height: 20px;
-  text-align: center;
-  color: black;
-  align-items: center;
-  justify-content: space-between;
-  white-space: normal;
-  padding-left: 100px;
-`;
-
-export const Subtitle = styled.div`
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  justify-content: space-between;
-  font-weight: normal;
-  font-size: 24px;
-  line-height: 15px;
+  font-size: large;
   text-align: center;
-  padding-top: 100px;
-  white-space: normal;
-  padding-left: 100px;
-`;
-
-export const SubComponentTitle = styled.div`
-  font-weight: bold;
-  font-size: 35px;
-  line-height: 20px;
-  text-align: left;
   color: black;
-  align-items: left;
-  margin-top: 50px;
-  justify-content: space-between;
-  width: 100%;
+  margin-top: 40px;
 `;
 
-export const SubComponent = styled.div`
+export const ElementContainer = styled.div`
   display: flex;
   flex-direction: column;
   align-items: flex-start;
-  justify-content: space-between;
   font-weight: normal;
-  text-align: left;
-  width: 100%;
-  height: 100%;
-  cursor: pointer;
+  margin-bottom: 90px;
+  margin-right: 40px;
```

Comment:
For a wide window size, 4 buttons show on the first row and 2 show on the second row. 
You can fix this by adding `flex: 0 0 26%;` here (26% makes sure that 4 buttons cannot fit in one row). This way the buttons should be spaced out without explicit right margin, so you can also delete `margin-right: 40px;`.